src/Akka.Streams.Kafka.Tests/Integration/CommitterFlowIntegrationTests.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Collections.Concurrent;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Akka.Configuration;
+using Akka.Streams.Dsl;
+using Akka.Streams.Kafka.Dsl;
+using Akka.Streams.Kafka.Messages;
+using Akka.Streams.Kafka.Settings;
+using Akka.Streams.TestKit;
+using Akka.Streams.Kafka.Helpers;
+using Confluent.Kafka;
+using Xunit;
+using Xunit.Abstractions;
+using Config = Hocon.Config;
+
+namespace Akka.Streams.Kafka.Tests.Integration
+{
+    public class CommitterFlowIntegrationTests : KafkaIntegrationTests
+    {
+        public CommitterFlowIntegrationTests(ITestOutputHelper output, KafkaFixture fixture) 
+            : base(nameof(CommitterFlowIntegrationTests), output, fixture)
+        {
+        }
+
+        [Theory]
+        [InlineData(1)]
+        [InlineData(5)]
+        public async Task CommitterFlow_commits_offsets_from_CommittableSource(int batchSize)
+        {
+            var topic1 = CreateTopic(1);
+            var topicPartition1 = new TopicPartition(topic1, 0);
+            var group1 = CreateGroup(1);
+
+            await GivenInitializedTopic(topicPartition1);
+
+            await Source
+                .From(Enumerable.Range(1, 100))
+                .Select(elem => new ProducerRecord<Null, string>(topicPartition1, elem.ToString()))
+                .RunWith(KafkaProducer.PlainSink(ProducerSettings), Materializer);
```

Comment:
There is a helpful `ProduceStrings` method in base class to make this in one line - you may want to use it here